eox_tenant/tenant_wise/__init__.py

```diff
@@ -26,25 +27,29 @@ def load_tenant_wise_overrides():
             )
 
         if allowed_proxies.get('TenantGeneratedCertificateProxy'):
-            certificate_list = [
-                'lms.djangoapps.certificates.models',
-                'lms.djangoapps.certificates.queue',
-            ]
-
             set_as_proxy(
-                modules=certificate_list,
                 model='GeneratedCertificate',
-                proxy=TenantGeneratedCertificateProxy
+                proxy=TenantGeneratedCertificateProxy,
+                djangoapp='lms.djangoapps.certificates',
             )
 
 
-def set_as_proxy(modules, model, proxy):
+def set_as_proxy(model, proxy, djangoapp='', modules=None):
     """
     Helper to patch a loaded module with a proxy object that has all the Tenant wise properties.
     """
-    if isinstance(modules, six.string_types):
-        modules = set([modules])
+    if djangoapp:
```

Comment:
Could it happen that there is no djangoapp? Why is the default '' ?

Reply:
the default is an empty string in order to indicate that the djangoapp value should be a string, and yes, there is no djangoapp when the method only is used to patch specific modules

Reply:
I cant exactly put my finger on it, but this functions looks weird.
Part of it is the empty string as default and the other part is that is has 2 different implementations behind the `if` statements.

Also, the djangoapp parameter does not have to be a djangoapp, does it? It is only a path to a package. Only that this time we expect this 'package' to have subpackages.